src/app/log-tool/log-tool-banner/log-tool-banner.component.css

```diff
@@ -24,4 +24,13 @@
     width: 45px;
     background-color: white;
     border-radius: 5px;
+}
+
+button{
+    border: none;
+    background-color: none;
```

Comment:
this background style isn't being applied. you should use classes instead of html tags in .css, you may need to use "!important" for this, the style your overriding is in app.components.css which I think ends up taking priority without it